src/dswx_sar/detect_inundated_vegetation.py

```diff
@@ -23,6 +23,11 @@ def run(cfg):
     processing_cfg = cfg.groups.processing
     outputdir = cfg.groups.product_path_group.scratch_path
     pol_list = processing_cfg.polarizations
+    pol_options = processing_cfg.polarimetric_option
+    
+    if pol_options is not None:
+        pol_list += pol_options
```

Comment:
Are `polarimetric_option`s necessary for any processing outside `pre_processing.run()`?

Reply:
the `polarimetric_option` are being used in `pre_processing`, `initial_threshold`, and `fuzzy_logic_compuation`. In the other steps, they are needed to get the filename. We may need to arrange the names of the intermediate files at some point, but currently we need it only for the filename.

Reply:
But we can delete the `ratio` method, because I believe this is useless in detecting water.